### ISSUE TITLE
How do you store commands?

### ISSUE THREAD

**Issue by danielo515** [ThePrimeagen/harpoon#250]
Sorry if the question is stupid but... how you define commands? Command 1, 2 etc the readme constantly references?
I can only see per project commands mentioned. Is that the only way?

**Comment by matu3ba** [comment]
> Is that the only way?

Yes, unless you hack up the internals. The config for all project is one huge json file.

**Comment by danielo515** [comment]
So there is no way to define "global" commands that are common to all the projects, right?
By the way, how is the project defined? I tried defining some commands, but they don't appear on the list of commands.

**Comment by matu3ba** [comment]
> By the way, how is the project defined?

You use the lua functions and bind your own commands. Use the ones given in the README.

> I tried defining some commands, but they don't appear on the list of commands.

You probably did not restart neovim or press `:so` on the file or you did not include them from your init.lua
You might want to pcall them, but first get it working.

I'd recommend to to ask these kind of questions on the discord or at least to provide the lua code of what you tried.
